### PR TITLE
FIX: a few react errors

### PR DESCRIPTION
**Pull request by limpbrains**
* a few require cycle errors
* move `navigation.setParams` from component constructor to `componentDidMount`

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/27auazpg0w9zuvr3w2zx1dr5fg

[download apk](http://filestorage.bluewallet.io:1488/1602578436.apk)

**Comment by marcosrdz** [comment]
I believe list.js, import.js need to be updated to conform to this change.

**Comment by limpbrains** [comment]
> I believe list.js, import.js need to be updated to conform to this change.

fixed, thanks!

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8x1q6b1xdq31wf36vme2yq2g3r

[download apk](http://filestorage.bluewallet.io:1488/1602657645.apk)

File: class/camera.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Linking, Alert } from 'react-native';
```
Comment (Overtorment) [inline class/camera.js:1]: feels like this file belongs in `blue_modules/`, not `class/`

**Review by Overtorment — APPROVED** [review]
lgtm

**Comment by Overtorment** [comment]
@marcosrdz your verdict

**Review by marcosrdz — APPROVED** [review]

**Pull request merged**